android/capacitor/src/main/java/com/getcapacitor/PluginCall.java

```diff
@@ -80,11 +86,28 @@ public void errorCallback(String msg) {
         this.msgHandler.sendResponseMessage(this, null, errorResult);
     }
 
+    /**
+     * @deprecated Use `reject()`.
+     */
     public void error(String msg, Exception ex) {
-        error(msg, null, ex);
+        reject(msg, null, ex);
     }
 
+    /**
+     * @deprecated Use `reject()`.
+     */
     public void error(String msg, String code, Exception ex) {
+        reject(msg, code, ex);
+    }
+
+    /**
+     * @deprecated Use `reject()`.
+     */
+    public void error(String msg) {
```

Comment:
Java allows to link the new method
Also the method should have the `@Deprecated` annotation, not just on the comment
```suggestion
    /**
     * @deprecated
     * Use {@link PluginCall#reject(String msg)} instead
     */
     @Deprecated
    public void error(String msg) {
```